src/QsCompiler/LlvmBindings/Extensions/LLVMMCJITCompilerOptionsExtensions.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) .NET Foundation and Contributors. All Rights Reserved. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace LlvmBindings.Interop
```

Comment:
If the package is named `Microsoft.Quantum.LlvmBindings.Native`, why isn't it the root namespace instead of just `LlvmBindings`? I believe that is the pattern for most of the packages that we build and distribute.

Reply:
The `LlvmBindings.Interop` is made up of generated code that comes from the LLVM pipeline and code that is manually authored here. Both that and the `LlvmBindings` namespace are considered an implementation detail and are not distributed externally. The `Microsoft.Quantum.LlvmBindings.Native` package doesn't even have any compiled .NET binaries; it just has native binaries for each platform and generated C#.